kalite/distributed/middleware.py

```diff
@@ -16,3 +18,23 @@ def process_request(self, request):
             and not request.path.startswith(settings.STATIC_URL)):
 
             raise PermissionDenied()
+
+
+class LogRequests:
+
+    def process_response(self, request, response):
+        logger = logging.getLogger('django.request')
+        # This is added because somehow requests aren't being logged
+
+        IGNORED_PATHS = (
```

Comment:
Are we going to be ok releasing 0.16 without log rotation? What are the implications for release?